recipes/dbus/1.x.x/conanfile.py

```diff
@@ -109,13 +109,18 @@ def generate(self):
             tc.project_options["launchd_agent_dir"] = os.path.join(self.package_folder, "res", "LaunchAgents")
         tc.project_options["x11_autolaunch"] = "enabled" if self.options.get_safe("with_x11", False) else "disabled"
         tc.project_options["xml_docs"] = "disabled"
+        tc.project_options["modular_tests"] = "enabled" if self.options.with_glib else "disabled" # glib is not found otherwise due to a buggy build.meson
```

Comment:
Can we get this fixed upstream?

Reply:
Agree with @jwillikers, we should at leats let upstream know :)